v2/dataplex/src/main/java/com/google/cloud/teleport/v2/options/DataplexJdbcIngestionOptions.java

```diff
@@ -250,4 +250,15 @@ public interface DataplexJdbcIngestionOptions
   Boolean getUseColumnAlias();
 
   void setUseColumnAlias(Boolean useColumnAlias);
+
+  @TemplateParameter.Integer(
+      order = 16,
+      optional = true,
+      description = "Set the data size going to be fetched and loaded in memory per Jdbc call.",
+      helpText =
+          "It should ONLY be used if the default value throws memory errors. If not set, using Beam's default "
+              + "fetch size.")
+  Integer getFetchSize();
```

Comment:
Should we set a default here? I see from the JdbcIO connector we have a default of 50,000: https://github.com/apache/beam/blob/81513f10d60f65c0873fc2484fb27a8e38fabc56/sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcIO.java#L373

Reply:
I intended to use JdbcIO's default the same as here: https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/1f37b59e49971e0b0c8c9864fbcee407e60aa588/v2/jdbc-to-googlecloud/src/main/java/com/google/cloud/teleport/v2/templates/JdbcToBigQuery.java#L157